docs/guides/javascript.md

```diff
@@ -55,14 +55,19 @@ Spectral comes with some rules and functions that are very specific to OpenAPI v
 ```js
 const { Spectral } = require('@stoplight/spectral');
 const { parseWithPointers } = require("@stoplight/yaml");
-const { oas3Functions, rules: oas3Rules } = require('@stoplight/spectral/dist/rulesets/oas3');
+const { oas3Functions } = require('@stoplight/spectral/dist/rulesets/oas3');
 
-
-const myOpenApiDocument = { ... };
+const myOpenApiDocument = {
+  // any parsed open api document
+};
 
 const spectral = new Spectral();
 spectral.addFunctions(oas3Functions());
-oas3Rules().then(rules => spectral.addRules(rules));
+spectral.loadRuleset('spectral:oas3')
+  .then(() => spectral.run(myOpenApiDocument))
+  .then(results => {
+    console.log('here are the results', results);
+  });
 ``` 
 
 _Spectral JS API does not yet support loading rulesets directly, but will be added in an upcoming version to avoid needing to load rules and functions separately._
```

Comment:
```suggestion
```

Reply:
Can you check the rest of this file to make sure nothing else is inaccurate.

Reply:
We still have `addRules` in a document covering custom rules
I'm not really sure how we could replace that example, as there is no way to do that without having another file with a ruleset.
The only way to have an equivalent solution would be to create a custom ruleset with a custom rule and load that ruleset.

Reply:
That''s OK! This PR is about adding the loadRuleset ability to catch JS up on the CLI, and we are swapping the common examples to show loadRuleset because it is preferred. We cannot totally remove addRules until custom functions are done, so this is all fine. 

When we do #430 we should make a new section in rulesets, and keep the addRules docs in here, but mark it as deprecated in the code _and_ in the docs. Then in v5.0 we remove it from both.